### PR TITLE
clients(devtools): bundle locale files

### PR DESCRIPTION
**Pull request by exterkamp**
**Summary**
Add locale files to devtools bundle.  Paving the way to localization inside devtools!

### CODE REVIEW

File: build/build-bundle.js [hunk 1]
```diff
@@ -74,8 +74,8 @@ async function browserifyFile(entryPath, distPath) {
     bundle.ignore(require.resolve('../lighthouse-core/report/html/html-report-assets.js'));
   }
 
-  // Don't include locales in DevTools or the extension for now.
```
Comment (patrickhulce) [inline build/build-bundle.js]: woah, like all of them? 😮

**Comment by Hoten** [comment]
image

but its fine b/c we get these files from the dt uploader (so we aren't really ballooning our footprint in Chrome download size)

File: build/build-bundle.js (hunk 1 above)
Comment (exterkamp) [reply build/build-bundle.js]: YUUUUUUP

🇦🇩🇦🇪🇦🇫🇦🇬🇦🇮🇦🇱🇦🇲🇦🇴🇦🇶🇦🇷🇦🇸🇦🇹🇦🇺🇦🇼🇦🇽🇦🇿🇧🇦🇧🇧🇧🇩🇧🇪🇧🇫🇧🇬🇧🇭🇧🇮🇧🇯🇧🇱🇧🇲🇧🇳🇧🇴🇧🇶🇧🇷🇧🇸🇧🇹🇧🇻🇧🇼🇧🇾🇧🇿🇨🇦🇨🇨🇨🇩🇨🇫🇨🇬🇨🇭🇨🇮🇨🇰🇨🇱🇨🇲🇨🇳🇨🇴🇨🇷🇨🇺🇨🇻🇨🇼🇨🇽🇨🇾🇨🇿🇩🇪🇩🇯🇩🇰🇩🇲🇩🇴🇩🇿🇪🇨🇪🇪🇪🇬🇪🇭🇪🇷🇪🇸🇪🇹🇫🇮🇫🇯🇫🇰🇫🇲🇫🇴🇫🇷🇬🇦🇬🇧🇬🇩🇬🇪🇬🇫🇬🇬🇬🇭🇬🇮🇬🇱🇬🇲🇬🇳🇬🇵🇬🇶🇬🇷🇬🇸🇬🇹🇬🇺🇬🇼🇬🇾🇭🇰🇭🇲🇭🇳🇭🇷🇭🇹🇭🇺🇮🇩🇮🇪🇮🇱🇮🇲🇮🇳🇮🇴🇮🇶🇮🇷🇮🇸🇮🇹🇯🇪🇯🇲🇯🇴🇯🇵🇰🇪🇰🇬🇰🇭🇰🇮🇰🇲🇰🇳🇰🇵🇰🇷🇰🇼🇰🇾🇰🇿🇱🇦🇱🇧🇱🇨🇱🇮🇱🇰🇱🇷🇱🇸🇱🇹🇱🇺🇱🇻🇱🇾🇲🇦🇲🇨🇲🇩🇲🇪🇲🇫🇲🇬🇲🇭🇲🇰🇲🇱🇲🇲🇲🇳🇲🇴🇲🇵🇲🇶🇲🇷🇲🇸🇲🇹🇲🇺🇲🇻🇲🇼🇲🇽🇲🇾🇲🇿🇳🇦🇳🇨🇳🇪🇳🇫🇳🇬🇳🇮🇳🇱🇳🇴🇳🇵🇳🇷🇳🇺🇳🇿🇴🇲🇵🇦🇵🇪🇵🇫🇵🇬🇵🇭🇵🇰🇵🇱🇵🇲🇵🇳🇵🇷🇵🇸🇵🇹🇵🇼🇵🇾🇶🇦🇷🇪🇷🇴🇷🇸🇷🇺🇷🇼🇸🇦🇸🇧🇸🇨🇸🇩🇸🇪🇸🇬🇸🇭🇸🇮🇸🇯🇸🇰🇸🇱🇸🇲🇸🇳🇸🇴🇸🇷🇸🇸🇸🇹🇸🇻🇸🇽🇸🇾🇸🇿🇹🇨🇹🇩🇹🇫🇹🇬🇹🇭🇹🇯🇹🇰🇹🇱🇹🇲🇹🇳🇹🇴🇹🇷🇹🇹🇹🇻🇹🇼🇹🇿🇺🇦🇺🇬🇺🇲🇺🇸🇺🇾🇺🇿🇻🇦🇻🇨🇻🇪🇻🇬🇻🇮🇻🇳🇻🇺🇼🇫🇼🇸🇾🇪🇾🇹🇿🇦🇿🇲🇿🇼

**Comment by exterkamp** [comment]
@paulirish know why Zeit is sad?

**Comment by brendankenny** [comment]
> but its fine b/c we get these files from the dt uploader

> 4.9M	lighthouse-dt-bundle.js

> fine

what would it take to setup a dependent remote module, @paulirish?

**Comment by patrickhulce** [comment]
image

There is something very ironic about shipping a ~5MB js bundle in a tool that complains about bundle sizes 😆 

I realize there's not an immediate better option available to us at the moment, but +1 to @brendankenny's idea if it's not a gargantuan task. We could theoretically ship a separate "locale rewritter" bundle that could be downloaded if the user wanted to switch to a different locale after the run. This would also defer our parsing costs for all these locales even if every user wanted this functionality.

Total aside, do we have stats on the locale data usage in PSI? Is it identical to country stats or is it based on preferred locale?

**Comment by Hoten** [comment]
> There is something very ironic about shipping a ~5MB js bundle in a tool that complains about bundle sizes 😆

This seems like a false equivalence to me. I think we needn't apply the same constraints of websites on a developer tool. i hold this pretty loosely just sayin'

But if the remote module thing isn't crazy complicated yeah we should def do it.

**Comment by patrickhulce** [comment]
> This seems like a false equivalence to me.

Oh for sure wasn't trying to equate them, there's just some irony there. If we were equating, we wouldn't have ever reached 1.6MB to begin with 😉

**Comment by paulirish** [comment]
Discussed on Slack, Edge DevTools team has some ideas for bundling these locale files in a different mechanism. As such we won't need to explode the -bundle.js file with all locales embedded.

**Pull request closed without merging**